cards/src/main/resources/cards/custom/group6/witch_doctor/secrets/secret_devils_deal.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Devil's Deal",
+  "baseManaCost": 2,
+  "type": "SPELL",
+  "heroClass": "ROSE",
+  "rarity": "EPIC",
+  "description": "Secret: When your champion is attacked, give it +5 Health.",

```

Comment:
Not sure if this is a legit card, it should either say `Restore #5 health` or `Give it 5 armor`. ask mr witch doctor

Reply:
<img width="842" alt="Screen Shot 2019-06-28 at 2 02 29 PM" src="https://user-images.githubusercontent.com/31571066/60371226-7c945a80-99ad-11e9-8ae6-6a785cb58591.png">